### PR TITLE
[cdc_stream] Merge cdc_stream into asset_stream_manager

### PR DESCRIPTION
**Pull request by ljusten**
Adds start and stop commands to asset_stream_manager.
asset_stream_manager will be renamed to cdc_stream next.

### CODE REVIEW

**Review by wurwunchik — APPROVED** [review]

File: asset_stream_manager/start_command.cc [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+StartCommand::StartCommand(int* exit_code)
+    : BaseCommand("start",
+                  "Start streaming files from a Windows to a Linux device",
+                  exit_code) {}
+
+StartCommand::~StartCommand() = default;
+
+void StartCommand::RegisterCommandLineFlags(lyra::command& cmd) {
+  verbosity_ = 2;
```
Comment (wurwunchik) [inline asset_stream_manager/start_command.cc:36]: It did not store my comments :(
So, again. Could you use a named constant here for `2`, please?

Comment (wurwunchik) [reply asset_stream_manager/start_command.cc:36]: (Here and below)

File: asset_stream_manager/stop_command.cc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+namespace cdc_ft {
+
+StopCommand::StopCommand(int* exit_code)
+    : BaseCommand("stop", "Stops a streaming session", exit_code) {}
+
+StopCommand::~StopCommand() = default;
+
+void StopCommand::RegisterCommandLineFlags(lyra::command& cmd) {
+  verbosity_ = 2;
```
Comment (wurwunchik) [inline asset_stream_manager/stop_command.cc:34]: The same here.

File: asset_stream_manager/start_command.cc [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+  ssh_port_ = 22;
+  cmd.add_argument(
+      lyra::opt(ssh_port_, "port")
+          .name("--ssh-port")
+          .help("Port to use while connecting to the remote instance being "
+                "streamed to, default: " +
+                std::to_string(ssh_port_)));
+
+  path::GetEnv("CDC_SSH_COMMAND", &ssh_command_).IgnoreError();
```
Comment (wurwunchik) [inline asset_stream_manager/start_command.cc:58]: What would happen if there is a variable (for example, outdated from previous runs) and the user defines a new one. Which will survive?

File: asset_stream_manager/start_command.cc (hunk 1 above)
Comment (ljusten) [reply asset_stream_manager/start_command.cc:36]: Yeah, I guess the diff shows the last PR plus this PR. This should go away once the last PR is merged. Please hold off reviewing this PR for now.

File: asset_stream_manager/start_command.cc (hunk 3 above)
Comment (ljusten) [reply asset_stream_manager/start_command.cc:58]: Not sure what you mean. Commands only run once, so there can't be a "previous" ssh_command_.

File: asset_stream_manager/stop_command.cc (hunk 2 above)
Comment (ljusten) [reply asset_stream_manager/stop_command.cc:34]: See above.

File: asset_stream_manager/start_command.cc (hunk 3 above)
Comment (ljusten) [reply asset_stream_manager/start_command.cc:63]: After talking offline, if there's an environment variable set and a command line flag, the command line flag will win.

**Review by wurwunchik — APPROVED** [review]

**Review by wurwunchik — APPROVED** [review]

**Pull request merged**